app/models/manageiq/providers/ibm_power_hmc/infra_manager/event_target_parser.rb

```diff
@@ -39,7 +41,11 @@ def parse
         # have changed (e.g. RMCState, PartitionName, PartitionState etc...)
         # This may be used to perform quick property REST API calls to the HMC
         # instead of querying the full LPAR data.
-        new_targets << {:assoc => :vms, :ems_ref => elems[:uuid]}
+        if elems[:uuid].eql?(NO_UUID_VALUE)
```

Comment:
No need for `eql?`, prefer a normal `==`

```suggestion
        if elems[:uuid] == NO_UUID_VALUE
```